std/string.d

```diff
@@ -5794,6 +5829,7 @@ unittest
 bool isNumeric(S)(S s, bool bAllowSep = false) if (isSomeString!S)
 {
     import std.algorithm.comparison : among;
+    import std.range.primitives : front, popFront;
 
```

Comment:
Hmm, it appears that `empty` is pulled in through the selective import at the top from `std.array`, which in turn publicly imports it from `std.range.primitives`.  Would you like me to turn the `empty` public import at the top into a direct import from `std.range.primitives` or leave it from `std.array`?